src/main/java/org/terasology/launcher/repositories/Jenkins.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.launcher.repositories;
+
+import java.util.Date;
```

Comment:
check run says this is unused ;)

Reply:
then it's probably right... :sweat_smile: